### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by JakariaJishan**
- [x] Use React Router.
- [x] Use styles in a React app.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
 function App() {
   return (
     <>
-      <Calculator />
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Home />} />
+          <Route path="calculator" element={<Calculator />} />
+          <Route path="quote" element={<Quote />} />
+          <Route path="*" element={<h1>Not found</h1>} />
```
Comment (VICTOR-CHUKWU) [inline src/App.js:16]: - [ ] nice to have a route that takes care of error page :+1:

File: src/components/navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+              <NavLink to="/">Home</NavLink>
+            </li>
+            <li>
+              <NavLink to="/calculator">Calculator</NavLink>
+            </li>
+            <li>
+              <NavLink to="/quote">Quote</NavLink>
+            </li>
+          </ul>
+        </nav>
```
Comment (VICTOR-CHUKWU) [inline src/components/navbar/Navbar.js:22]: - [ ] good use of react router :clap:

File: src/components/quote/Quote.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          {quote.content}
+          &rdquo; -
+          {' '}
+          {quote.author}
+        </blockquote>
+      ) : (
+        <p>Loading ....</p>
+      )}
+    </div>
+  );
```
Comment (VICTOR-CHUKWU) [inline src/components/quote/Quote.js:29]: - [ ] I like the loading displayed while waiting for the quotes to load, good job :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi <STUDENT_MENTION>, 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JakariaJishan** [comment]
thank you so much, this kind of little comment inspires me.❤️

**Pull request merged**